### ISSUE TITLE
Enhancement Request: Add extra_args to rdp config options

### ISSUE THREAD

**Issue by dcode** [hashicorp/vagrant#13122]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I forget to add ` -- /cert-ignore` when I run `vagrant rdp` and `xfreerdp` silently quits.

**Describe the solution you'd like**
I understand there's different arguments for different RDP clients, but it would save me a whole lot of frustration if I could add to my local Vagrantfile something like `config.rdp.extra_args = ["/cert-ignore", "/dynamic-resolution"]`, to allow RDP to connect and later to resize as my window resizes.

**Describe alternatives you've considered**
I've considered making a bash function like `vagrant-rdp`, which is totally doable, but I'd rather just use 1st class vagrant commands.

**Additional context**

**Comment by 3ximus** [comment]
I agree, this would be pretty helpful. Doesn't seem to complicated to add. I might submit a PR soon when I find the time
